### PR TITLE
Fix/wrong id type use as attrs #1124 

### PR DESCRIPTION
**Pull request by AlvaroVega**
clean PR version of https://github.com/telefonicaid/iotagent-node-lib/pull/1125

issue https://github.com/telefonicaid/iotagent-node-lib/issues/1124

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "prettier": "prettier --config .prettierrc.json --write '**/**/**/**/*.js' '**/**/**/*.js' '**/**/*.js' '**/*.js' '*.js'",
     "prettier:text": "prettier 'README.md' 'doc/*.md' 'doc/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
     "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
-    "test:expression": "nyc --reporter=text mocha --recursive 'test/unit/expressions/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
+    "test:expression": "nyc --reporter=text mocha --recursive 'test/unit/ngsiv2/plugins/multientity-plugin_test.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```
Comment (fgalan) [inline package.json:40]: The target name (test:expression) missmatches the name of the .js file... Maybe this change should be rolled-back?

```suggestion
    "test:expression": "nyc --reporter=text mocha --recursive 'test/unit/expressions/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```

File: test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin1.json [hunk 2]
```diff
@@ -17,11 +15,11 @@
       "TimeInstant": {
         "type": "DateTime",
         "value": "2016-05-30T16:25:22.304Z"
-      }
+      },
+      "id": "ws4",
+      "type": "WeatherStation"
```
Comment (fgalan) [inline test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin1.json:20]: Is this change in the order of the keys in the JSON needed to make the tests using this .json as expectation to work?

A bit surprising... as the key order in the JSON files shouldn't matter...

Comment (AlvaroVega) [reply test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin1.json:20]: It matters for the way of tests is checking that json fields, just if both strings are equivalent.

Comment (fgalan) [reply test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin1.json:20]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Review by mrutid — APPROVED** [review]
LGTM

**Pull request merged**